Jenkinsfile

```diff
@@ -598,6 +598,19 @@ pipeline {
                         buildHipClangJobAndReboot(setup_cmd: "", build_cmd: "", execute_cmd: execute_cmd, needs_gpu:false, needs_reboot:false)
                     }
                 }
+                stage('Check GTest Format') {
+                agent { label rocmnode("nogpu") }
+                when {
+                    changeset "**/test/utils/**"
+                }
```

Comment:
Will this only run if changes happened inside test/utils? 

I think we would want this to run if changes happened in  changeset "**/test/gtest/**"

Also, did we run it generally on the latest changes to ensure everything is now passing?